chrome/content/helpers.js

```diff
@@ -0,0 +1,76 @@
+var isNode = (typeof process === "object");
+var isJpm = !isNode && (typeof require === "function");
+
+var OS;
+
+if (!isNode) {
+  var Ci, Cu, Cc;
+  if (isJpm) {
+    ({ Ci, Cu, Cc } = require("chrome"));
+  } else {
+    ({ interfaces: Ci, utils: Cu, classes: Cc } = Components);
+    Cu.import("resource:///modules/imServices.jsm");
+    Cu.import("resource:///modules/imXPCOMUtils.jsm");
+  }
+  Cu.import("resource://gre/modules/NetUtil.jsm");
+  Cu.import("resource://gre/modules/FileUtils.jsm");
+  Cu.import("resource://gre/modules/osfile.jsm");
+}
+
+var helpers = {
+  profilePath: function(filename) {
+    return isNode ?
+      path.resolve(__dirname, filename) :
+      OS.Path.join(OS.Constants.Path.profileDir, filename);
+  },
+
+  getAccounts: function* () {
+    let accounts = Services.accounts.getAccounts();
+    while (accounts.hasMoreElements()) {
+      yield accounts.getNext();
+    }
+  },
+
+  readTextFile: function(filename) {
```

Comment:
Should we also use the promisified `OS.File.read` (https://developer.mozilla.org/en-US/docs/Mozilla/JavaScript_code_modules/OSFile.jsm/OS.File_for_the_main_thread) instead?